.github/workflows/ci.yml

```diff
@@ -16,4 +16,4 @@ jobs:
     secrets: inherit
     with:
       repo-name: module-ballerinax-candid
-      additional-test-flags: -x test
+      additional-test-flags: -Pgroups=mock
```

Comment:
We can use a single test suite for both mock and live test cases like here.
https://github.com/ballerina-platform/module-ballerinax-twilio/tree/master/ballerina/tests

In that way, we don't need to use `-Pgroups=mock` in `additional-test-flags`